src/openassetio-core-c/private/managerAPI/CManagerInterface.cpp

```diff
@@ -13,15 +13,17 @@ namespace managerAPI {
 
 constexpr size_t kStringBufferSize = 500;
 
+// Buffer for error message. Statically allocated per-thread to avoid
+// the need to reallocate on ever function call.
+static thread_local char errorMessageBuffer[kStringBufferSize];
```

Comment:
[question] Is this really a significant cost at the moment? To make it worth the risk of accidentally coupling test data?

Reply:
Reverted in ffac5bd2 ready to be squashed out of existence, and updated `info` to use a stack-allocated buffer in cf73708f